### DISCUSSION TITLE
Problem unselecting rows on DataTable

### DISCUSSION THREAD

**Discussion by bhaidar** [primefaces/community#1330] [PrimeVue]
Hi,

I am trying to add row-checkbox for my lazy loading datatable with no luck.

On the demo page, it seems that unselecting all records doesn't do anything:
https://stackblitz.com/edit/hjvucs?file=src%2FApp.vue

The other problem I am facing locally, I can select a row, but I then cannot unselect. It remains selected. Is there a solution for that?

Thanks
Bill

**Comment by bhaidar** [comment]
Hey @cagataycivici I appreciate someone checking this out. The online demo is not working and has issues with checking/unchecking checkboxes. This is really a much needed feature especially when using Lazy Loading.


Thanks
Bilal

**Comment by tugcekucukoglu** [comment]
Can not replicate the problem on the showcase. https://primevue.org/datatable/#dtproducts

**Reply by bhaidar** [reply]
Here you go:
https://stackblitz.com/run?file=src%2FApp.vue


try selecting all and then try to unselect.
